sys/include/byteorder.h

```diff
@@ -388,13 +389,6 @@ static inline uint64_t ntohll(uint64_t v);
 
 /* **************************** IMPLEMENTATION ***************************** */
 
-#ifdef HAVE_NO_BUILTIN_BSWAP16
-static inline unsigned short __builtin_bswap16(unsigned short a)
-{
-    return (a << 8) | (a >> 8);
-}
-#endif
```

Comment:
For compatibility with GCC version 3 or so?

Anyway, even the AVR GCC compiler shipped in Debian / Ubuntu supports this, and that is some GCC 4.X compiler.

(We do in fact have that one in the CI. This will ensure we won't move too fast.)

Reply:
I see, thanks!